__tests__/context/throw.test.js

```diff
@@ -32,21 +32,6 @@ describe('ctx.throw(err)', () => {
   })
 })
 
-describe('ctx.throw(err, status)', () => {
-  it('should throw the error and set .status', () => {
-    const ctx = context()
-    const error = new Error('test')
-
-    try {
-      ctx.throw(error, 422)
-    } catch (err) {
-      assert.strictEqual(err.status, 422)
-      assert.strictEqual(err.message, 'test')
-      assert.strictEqual(err.expose, true)
-    }
-  })
-})
-
```

Comment:
[The signature now should in the following order: status, errorOrMessage, properties.](https://github.com/jshttp/http-errors/tree/master?tab=readme-ov-file#createerrorstatus-error-properties)